apps/drec-api/src/pods/issuer/issuer.service.ts

```diff
@@ -84,17 +85,18 @@ export class IssuerService {
       ownerGroupedDevices.flatMap(async (ownerBasedGroup: IDevice[], i) => {
         values(groupBy(ownerBasedGroup, deviceGroupRule2)).map(
           async (countryBasedGroup: IDevice[], j) => {
-            const categorizedGroup: IDeviceGroup = {
-              id: 0,
-              name: `Default Group ${ownerBasedGroup[i]?.organizationId}_${i}-${j}`,
-              organizationId: ownerBasedGroup[i]?.organizationId,
-              devices: countryBasedGroup,
-            };
-            return await this.issueCertificateForGroup(
-              categorizedGroup,
-              startDate,
-              endDate,
-            );
+            // const categorizedGroup: IDeviceGroup = {
```

Comment:
This part was commented for now as it won`t make sense anymore and it will be replaced in the future tasks.
For now I left it as commented in case I might need parts from it.